prisma/migrations/20250305095815_add_person_and_anniversary_models/migration.sql

```diff
@@ -0,0 +1,89 @@
+/*
+  Warnings:
+
+  - The primary key for the `Feedback` table will be changed. If it partially fails, the table could be left without primary key constraint.
+  - You are about to drop the column `feedbackID` on the `Feedback` table. All the data in the column will be lost.
+  - You are about to drop the column `feedbackUUID` on the `Feedback` table. All the data in the column will be lost.
+  - A unique constraint covering the columns `[uuid]` on the table `Feedback` will be added. If there are existing duplicate values, this will fail.
+  - The required column `uuid` was added to the `Feedback` table with a prisma-level default value. This is not possible if the table is not empty. Please add this column as optional, then populate it before making it required.
+
+*/
+-- DropForeignKey
+ALTER TABLE "Gift" DROP CONSTRAINT "Gift_userUUID_fkey";
+
+-- DropIndex
+DROP INDEX "Feedback_feedbackUUID_key";
+
+-- AlterTable
+ALTER TABLE "Feedback" DROP CONSTRAINT "Feedback_pkey",
+DROP COLUMN "feedbackID",
+DROP COLUMN "feedbackUUID",
+ADD COLUMN     "id" SERIAL NOT NULL,
+ADD COLUMN     "uuid" TEXT NOT NULL,
+ADD CONSTRAINT "Feedback_pkey" PRIMARY KEY ("id");
```

Comment:
Oliko tämä automaattigeneroitu? Ei nimittäin toimi, jos tietokannassa on yksikin palaute, koska `ADD COLUMN     "id" SERIAL NOT NULL` lisää sarakkeen joka ei saa olla `NULL` mutta joka olisi `NULL` vanhoille riveille -> virhe.  Sarakkeet kannattaa uudelleen nimetä näin:

```sql
ALTER TABLE table_name
RENAME COLUMN old_column_name TO new_column_name;
```

Kannattaa muuten kokeilla lopullista migraatioita tietokantaan, jossa on jotain dataa.